src/components/expression/expression.js

```diff
@@ -79,11 +86,13 @@ genotet.ExpressionView = function(viewName, params) {
 
   // Set up data loading callbacks.
   $(this.container).on('genotet.ready', function() {
+    this.data.tfa.fileName = params.tfaFileName;
```

Comment:
BTW this is a separate issue from the theme of this pullreq. So if you like you can fix the tfa/matrix dependencies issue in another pullreq.